test.js

```diff
@@ -559,7 +559,7 @@ function runTest (client, db) {
           instance.storeRetained(packet, function (err) {
             t.notOk(err, 'no error')
 
-            setTimeout(checkRetained.bind(this), 5000) // https://docs.mongodb.com/manual/core/index-ttl/#timing-of-the-delete-operation
+            setTimeout(checkRetained.bind(this), 65000) // https://docs.mongodb.com/manual/core/index-ttl/#timing-of-the-delete-operation
```

Comment:
This will make the tests take 65s. This is unacceptable for a dev environment.

You can use docker to start the container, it should be feasible.

Reply:
@mcollina That was already present in test until my fix some weeks ago when I fixed them. Anyway, I will fix it